src/Integrations/Integrations/Mysqli/MysqliIntegration.php

```diff
@@ -40,20 +43,30 @@ public function init()
         $integration = $this;
 
         \DDTrace\trace_function('mysqli_connect', function (SpanData $span, $args, $result) use ($integration) {
-            list($host) = $args;
+            list($host,,, $dbName) = $args;
```

Comment:
You cannot use list() with optional args, this will give you an undefined index warning if the arg isn't given (under DD_TRACE_DEBUG=1). Needs `!empty($args[3])` directly.

Reply:
Good catch! thank you